### DISCUSSION TITLE
Unexpected refreshing of inlay hints

### DISCUSSION THREAD

**Discussion by failable** [helix-editor/helix#6598] [General]
The inlay hint is refreshed after some motions like `g g` or `g e`, while this does not happen for some other motions like `g d` or `ctrl-o`.

**Comment by pascalkuthe** [comment]
Inlay hints are always refreshed when you scroll a certain distance that's just fundamentally how tey work. With gd and c-o you likely went to a different file (or to a very close location in the same file) where refreshing was not possible. In the future we want to refresh inlay hints quicker on scrolilling/positom changes and slower on edits so it will be less noticable but that requires some larger work (introducing an event system) so it will take a while to address.

Fundamentalky this is intended behavior tough and just howninlay hints work. I just eant to make the timeout a bit less noticeable
